lib/ClientConfig.cpp

```diff
@@ -1,7 +1,7 @@
 #include "ClientConfig.h"
 
 #include "swss/logger.h"
-#include "swss/json.hpp"
```

Comment:
will that json.hpp will be removed from sonic-swss-common repo ?

Reply:
Yes, it will be removed soon.